app/controllers/idv/phone_controller.rb

```diff
@@ -225,8 +225,8 @@ def formatted_previous_phone_step_params_phone
 
     def gpo_letter_available
       return @gpo_letter_available if defined?(@gpo_letter_available)
-      @gpo_letter_available ||= FeatureManagement.gpo_verification_enabled? &&
-                                !Idv::GpoMail.new(current_user).rate_limited?
+      policy = Idv::GpoVerifyByMailPolicy.new(current_user)
+      @gpo_letter_available ||= policy.send_letter_available?
```

Comment:
I do not think we want `||=` here, I think with the guard above we can do a direct assignment
```suggestion
      @gpo_letter_available = policy.send_letter_available?
```

Reply:
Nice catch -- I had just changed what was there and missed that this was unnecessary. Will update these.